prep-javascript-variables/index.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>JavaScript Variables</title>
+</head>
+<body>
+  <script>
+    var fullName = 'Jonathan Matsuda';
+    console.log(fullName);
+    var numberOfPets = 0;
+    console.log(numberOfPets);
+    var likesVegetables = true;
+    console.log(likesVegetables);
+    var nothing = 'null';
```

Comment:
Hi Jonathan, you can leave off the quotes so that the JavaScript engine knows you mean `null` and not a string.